CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 ### Added
 
 - Added support for Azure SDK traces instrumentation on .NET Framework.
+- Added support for `WCFCLIENT` instrumentation on .NET.
```

Comment:
Should we also update what is removed as a part of this PR?

Reply:
I think that we should call out the switch from custom byte code instrumentation to the contrib library.

Reply:
You are right, when I look back at the changelog updates for WCF I think that we do not need to call out the switch. We are already calling out what should be the only noticeable change. Even with this PR the current solution relies on both byte-code instrumentation and the contrib instrumentation library, so we are not eliminating the byte code instrumentation.